common/variant/src/main/java/org/apache/spark/variant/MalformedVariantException.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.variant;
+
+import scala.collection.immutable.Map$;
+
+import org.apache.spark.QueryContext;
+import org.apache.spark.SparkRuntimeException;
+
+public class MalformedVariantException extends SparkRuntimeException {
```

Comment:
Unless we need to catch this certain exception to do something, it's recommended to simply use `SparkException` or `SparkRuntimeException` (if unchecked exception is needed) with error classes.

Reply:
end users should distinguish errors by error class, not different java exception class.

Reply:
Makes sense. I don't think we need to catch this certain exception.